ibis-server/tests/routers/v3/connector/postgres/test_query.py

```
@@ -525,3 +539,50 @@ async def test_rlac_query(client, manifest_str, connection_info):
     result = response.json()
     assert len(result["data"]) == 1
     assert result["data"][0][0] == "Customer#000000001"
+
+
+async def test_clac_query(client, manifest_str, connection_info):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)